src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -3,5 +3,5 @@
 import core.basesyntax.model.User;
 
 public interface RegistrationService {
-    User register(User user);
+    User register(User user) throws UserRegistrationException;
```

Comment:
```suggestion
    User register(User user);
```
<img width="795" alt="Screen Shot 2022-12-29 at 1 42 22 PM" src="https://user-images.githubusercontent.com/90692308/209946380-bb554b91-00fd-47ba-bf99-3d67ee8ffad2.png">
Ментор же пояснив чому цього не треба робити